w12h02/test/pgdp/ds/MultiStackSyncTests.java

```diff
@@ -0,0 +1,92 @@
+package pgdp.ds;
+
+import org.junit.jupiter.api.Test;
+import pgdp.ds.MultiStack;
```

Comment:
This is obsolete now that you changed the package. (leads to compiler error)

```suggestion
```

Reply:
The way Guthub works this suggestion will overwrite package